e2e_tests/docker_image/parsec-service-test-all.Dockerfile

```diff
@@ -84,19 +84,20 @@ RUN cd nanopb-0.4.4-linux-x86 \
 	&& make install
 RUN rm -rf nanopb-0.4.4-linux-x86 nanopb-0.4.4-linux-x86.tar.gz
 
-# Install mock Trusted Services
 # Setup git config for patching dependencies
 RUN git config --global user.email "some@email.com"
 RUN git config --global user.name "Parsec Team"
+# Install an old version mock Trusted Services compatible with old parsec 0.7.0
+# used in generate_key.sh script
 RUN git clone https://git.trustedfirmware.org/TS/trusted-services.git --branch integration \
 	&& cd trusted-services \
-	&& git reset --hard 389b50624f25dae860bbbf8b16f75b32f1589c8d
+	&& git reset --hard 35c6d643b5f0c0387702e22bf742dd4878ca5ddd
 # Install correct python dependencies
 RUN pip3 install -r trusted-services/requirements.txt
 RUN cd trusted-services/deployments/libts/linux-pc/ \
 	&& cmake . \
 	&& make \
-	&& cp libts.so* nanopb_install/lib/libprotobuf-nanopb.a mbedtls_install/lib/libmbedcrypto.a /usr/local/lib/
+	&& cp libts.so* nanopb_install/lib/libprotobuf-nanopb.a mbedcrypto_install/lib/libmbedcrypto.a /usr/local/lib/
 RUN rm -rf trusted-services
```

Comment:
I'd suggest moving this to the `generate-keys.sh` script as well, no point having it twice here and it makes it somewhat confusing as to what's actually running.